react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import { Header } from './Header';
+import { ImageContainer } from './ImageContainer';
+import { ImageCaption } from './ImageCaption';
+import { ImageDescription } from './ImageDescription';
+import { ButtonContainer } from './ButtonContainer';
+import './App.css';
+
+const headerText = 'React Image Bank';
+const imageUrl = '/starry-sky.jpeg';
+const imageCaption = 'An overrated image of space';
```

Comment:
You think it's overrated, huh? I think I agree. There are numerous much more beautiful space images.